src/database/models/userProfile.ts

```diff
@@ -0,0 +1,138 @@
+import { DataTypes, Model } from 'sequelize'
+import sequelizeConnection from '../config/db.config'
+import User from '../models/userModel'
+
+/**
+ * User Interface
+ */
+export interface UserProfileAttributes {
+  id: number
+  userId: number
+  username: string
+  email: string
+  birthdate: string
+  gender: string
+  preferredLanguage: string
+  preferredCurrency: string
+  phoneNumber: string
+  homeAddress: string
+  billingAddress: string
+  profileImageUrl: string
+  createdAt: Date
+  updatedAt: Date

```

Comment:
You can also make these optional, cause sometimes a user may not have filled out all out the